src/mumble/AudioInput.h

```diff
@@ -210,7 +210,9 @@ class AudioInput : public QThread {
 
 	bool bEchoMulti;
 	Settings::NoiseCancel noiseCancel;
+	// Standardized microphone sample rate (samples/s)
 	static const unsigned int iSampleRate = SAMPLE_RATE;
+	/// Input data is divided into frames, each of which is 10ms of audio data
```

Comment:
I think at this point it'd be nice to include the sample calculation that I did here on GitHub in the comment. That way, it should be more clear why this should result in 10 ms of audio